File: 75-sort-colors/sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int n=nums.size();
        for(int i=0;i<n;i++){
            int mini=i;
            for(int j=i+1;j<n;j++){
                if(nums[mini]>nums[j]){
                    mini=j;
                }
            }
            swap(nums[mini],nums[i]);
        }
    }
};

File: 75-sort-colors/sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        for i in range(len(nums)):
            mini=i
            vis=False
            for j in range(i+1,len(nums)):
                if nums[mini]>nums[j]:
                    mini=j
                    vis=True
            if vis:
                nums[i],nums[mini]=nums[mini],nums[i]
        
        